examples/notebooks/beam-ml/run_inference_tensorflow.ipynb

```diff
@@ -313,14 +350,34 @@
       "metadata": {
         "id": "2JbE7WkGcAkK"
       },
-      "execution_count": 8,
+      "execution_count": 18,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Save the weights."
```

Comment:
Maybe something along the lines of: `Instead of saving the model, you can also save/load a model using just the model weights and class. This is a smaller, more efficient way to represent your model.`

Basically, I want something to justify to the user why we're having them save things 2 ways so they know that in reality they just have to pick one.

Reply:
Done